web/modules/custom/asset_release/src/Controller/AssetReleaseController.php

```diff
@@ -122,29 +121,7 @@ public function overview(RdfInterface $rdf_entity) {
     $releases = array_filter($releases, function ($release) {
       return $release->access('view');
     });
-
-    // Retrieve all standalone distributions for this solution. These are
-    // downloads that are not associated with a release.
-    // The standalone distributions are not flagged in any way, and the relation
-    // between releases and distributions is backwards (the relation is present
-    // on the parent entity). We work around this by retrieving all
-    // distributions excluding the ones that are associated with a release.
-    $release_distribution_ids = [];
-    foreach ($releases as $release) {
-      foreach ($release->get('field_isr_distribution')->getValue() as $distribution_reference) {
-        $release_distribution_ids[] = $distribution_reference['target_id'];
-      }
-    }
-
-    $query = $this->queryFactory->get('rdf_entity');
-    $query
-      ->condition('rid', 'asset_distribution')
-      ->condition(OgGroupAudienceHelperInterface::DEFAULT_FIELD, $rdf_entity->id());
```

Comment:
Please, check https://github.com/ec-europa/joinup-dev/pull/1238/files#diff-c379272bb9dad959c9312d24c58f63f2L182 for the reason why this line was not working properly but was appearing to work properly on our tests.

I am pretty sure that manual check will not work when attempting to test this though I don't want to spend the time to check it.

The steps to reproduce would be to:
* Create a solution.
* Create a release of that solution on a draft state.
* Create a distribution of that release.
* Log out, because a facilitator/moderator has view access to the draft releases.
* Go to the "Download releases" page.

*Expected behaviour:* The distribution is hidden due to the parent release being hidden.
*Actual behaviour:* The distribution is displayed as a standalone distribution.
*Reason:* If you pay close attention to the code above, the releases are loaded but removes from the list of the `$releases` variable all those that the user does not have view access. Then, standalone distributions are attempted to be retrieved by loading all distributions belonging to the solution group apart from those that belong to a release. The later list is built excluding distributions belonging to the `$releases` variable above which though now does not include the draft releases. That means that all distributions belonging to draft releases are loaded normally and marked as standalone. This was the 3rd failure of this PR.